addons/event_sale/models/sale_order.py

```diff
@@ -174,12 +169,14 @@ def _get_sale_order_line_multiline_description_sale(self):
 
     def _get_display_price(self):
         if self.event_ticket_id and self.event_id:
-            # FIXME this is strange
-            # price_reduce is the price after discount
-            # shouldn't we leave the discount computation to sale
-            # and use the non reduced price here (aka price field)
```

Comment:
I did not change the behavior with the discount here, as it was done in the commit I backport because I wanted to limit the changes to the reported bug (everything is handled in the `price_reduce` if I understood correctly). I only added the currency conversion that was missing.

Reply:
Oh, maybe I misunderstood your comment. Are you asking me _not_ to remove the FIXME comment? It's true that I don't tackle the issue here

Reply:
Oh no I thought you were introducing this change in the commit. If it's removed from the original I guess it's fine.
Apparently this was introduced as part of a fix, I'm not sure how well it behaves though, it does seem weird.
https://github.com/odoo/odoo/pull/70856

You can leave this as is, I'll just make note of it :)

Reply:
Well, the comment has not been resolved in this version or PR, but was fixed in later ones (#93818 & fw-ports), so not a big problem if we remove it here :ok_hand:.